deployments/charts/kuma/templates/egress-deployment.yaml

```diff
@@ -96,12 +97,37 @@ spec:
             timeoutSeconds: 3
           resources: {{ toYaml .Values.egress.resources | nindent 12 }}
           volumeMounts:
+{{- if not .Values.egress.automountServiceAccountToken }}
+            - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+              name: serviceaccount-token
+              readOnly: true
+{{- end }}
             - name: control-plane-ca
               mountPath: /var/run/secrets/kuma.io/cp-ca
               readOnly: true
             - name: tmp
               mountPath: /tmp
       volumes:
+{{- if not .Values.egress.automountServiceAccountToken }}
+        - name: serviceaccount-token
+          projected:
+            defaultMode: 420
```

Comment:
isn't this a bit weird? read for the owner write for the group?

Reply:
That's decimal of 644 (see: https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/#tokenrequest-api)